internal/providers/terraform/aws/testdata/acmpca_certificate_authority_test/acmpca_certificate_authority_test.golden

```diff
@@ -0,0 +1,19 @@
+
+ Name                                                     Monthly Qty  Unit      Monthly Cost 
+                                                                                              
+ aws_acmpca_certificate_authority.private_ca                                                  
+ ├─ Private certificate authority                                   1  months         $400.00 
```

Comment:
Should there be a test case for a ca with usage privded?

Reply:
The following two resources have usage values.